pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -27,6 +27,9 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+#[cfg(any(feature = "try-runtime", test))]
+mod try_state;
```

Comment:
I would suggest you include `try_state` only with the feature on, and then feature-gate functions or modules inside of it. That's the cleaner approach and will probably also result in less lint errors being thrown around.

Reply:
I'm having trouble understanding your point. What exactly do you mean?

Reply:
Don't feature gate the mod with `any()`, but simply with `feature = "try-runtime"`. Then in the module you can export test-only functions.